app/util/createFormattedDate.js

```diff
@@ -0,0 +1,36 @@
+export const createFormattedDate = () => {
```

Comment:
While this looks fine to be is there a reason we arent just using moment js here?